cmp/grid/columns/Column.js

```diff
@@ -230,14 +235,14 @@ export class Column {
         const {renderer, elementRenderer} = this;
         if (renderer) {
             ret.cellRenderer = (agParams) => {
-                return renderer(agParams.value, {record: agParams.data, column: this, agParams});
+                return renderer(agParams.value, {record: agParams.data, column: this, agParams, gridModel});
```

Comment:
great!  Could also enhance the "metadata" doc strings below to indicate 'column, record, and gridModel'